_layouts/workshop_post.html

```diff
@@ -5,8 +5,7 @@
 	<header>
 		
 		{% include nav.html %} 	
-	
-		
+
     <style>	
```

Comment:
Can you also remove this <style> section (and the .css rules inside) in this file? Because it is not being used :thinking: